docs/reference/search/search-your-data/paginate-search-results.asciidoc

```diff
@@ -362,7 +362,7 @@ Perl::
 
 Python::
 
-    See https://elasticsearch-py.readthedocs.org/en/master/helpers.html[elasticsearch.helpers.*]
+    See https://elasticsearch-py.readthedocs.io/en/v8.14.0/helpers.html[elasticsearch.helpers.*]
```

Comment:
The correct URL here is `https://elasticsearch-py.readthedocs.io/en/` - this way it won't go stale when 8.15 is released etc. Also we want to merge to main and backport as necessary.